RELEASE_NOTES.md

```diff
@@ -1,5 +1,16 @@
 # Release Notes
 
+### Minor Changes
+* Change ROLE_ACB_ADMIN to ROLE_ACB
+* Change ROLE_ATL_ADMIN to ROLE_ATL
+* Remove ROLE_ACB_STAFF
+* Remove ROLE_ATL_STAFF
+
+### Data changes
+* Mark deprecated upload templates as deleted
```

Comment:
This part is going to have to wait until January 1st; the ACBs won't be ready for it until then